blog.md

```diff
@@ -1,26 +1,78 @@
 
 ## Infrastructure
 
-1. [OpenStack](http://www.openstack.org/) was installed across a set of physical machines using [Ursula](https://github.com/blueboxgroup/ursula).
-2. A 3 node [DEIS](http://deis.io/)
- cluster was then installed on [OpenStack](http://www.openstack.org/) using the terraform instructions [here](https://github.com/paulczar/deis/tree/openstack_provision_script/contrib/openstack).
-3. A standalone coreos node was created using [Terraform](https://terraform.io/) 
-4. A self replicating database was deployed on the [DEIS](http://deis.io/) nodes using the [paulczar/percona-galera](https://github.com/paulczar/docker-percona_galera) docker image which uses [etcd](https://coreos.com/etcd/) to find each other and form a cluster.
-5. A mysql proxy was deployed on the standalone node using the [paulczar/maxscale](https://hub.docker.com/r/paulczar/maxscale/) image which discovers the databases via [etcd](https://coreos.com/etcd/).
-6. [Jenkins](http://jenkins-ci.org/) was deployed to standalone node using the standard [jenkins docker image](https://hub.docker.com/_/jenkins/).
+While an IaaS is not strictly necessary here (I could run Deis straight on bare metal) it makes sense to use something like [OpenStack](http://www.openstack.org/) as provides excellent APIs to spin up VMs.  I installed OpenStack across across a set of physical machines using [Blue Box's](https://www.blueboxcloud.com/) [Ursula](https://github.com/blueboxgroup/ursula).
+
+Next the PaaS itself, I have familiarity with [Deis](http://Deis.io/) already and I really like its (Heroku-esque) user experience.  I span up a three node Deis cluster on OpenStack using the terraform instructions [here](https://github.com/paulczar/Deis/tree/openstack_provision_script/contrib/openstack).
+
+I also span an additional three [CoreOS](https://coreos.com) nodes using [Terraform](https://terraform.io/) on which I ran [Jenkins](http://jenkins-ci.org/) using the standard [jenkins docker image](https://hub.docker.com/_/jenkins/).
+
+To demonstrate a backing service for my application I also ran a Percona database cluster (one on each of the CoreOS nodes) using the [paulczar/percona-galera](https://github.com/paulczar/docker-percona_galera) docker image which uses [etcd](https://coreos.com/etcd/) to find each other and form automatically create the galera replication set. I also deployed a mysql proxy using the [paulczar/maxscale](https://hub.docker.com/r/paulczar/maxscale/) image which discovers the databases via [etcd](https://coreos.com/etcd/).
+
+## Ghost
+
+The application I chose to demo is the [Ghost blogging platform](https://ghost.org/download/) (literally the blog you are reading right now). I chose it because it's a fairly simple app with well-known backing service (MySQL). The source including my `Dockerfile` and customizations can be found in the [paulczar/ci-demo](https://github.com/paulczar/ci-demo) github repository.
```

Comment:
Note that Sysadvent is published on the Blogger platform, so there's a very good chance that they won't be reading it on your demo site directly.
« [...] The source**,** including my Dockerfile and customizations**,** [...] **G**ithub repository. »
* Added punctuation.
* Capitalised a name.